cfbs/utils.py

```diff
@@ -130,23 +130,21 @@ def strip_left(string, beginning):
     return string[len(beginning) :]
 
 
-def strip_right_any(string, suffixes: tuple):
-    if not string.endswith(suffixes):
-        return string
-
+def strip_right_any(string, suffixes):
```

Comment:
Why remove the type? In this case, it looks like the most specific type is;

```suggestion
def strip_right_any(string: str, suffixes: Iterable[str]) -> str:
```